docs/tutorials/motion/plan-motion-with-arm-gripper.md

```diff
@@ -0,0 +1,561 @@
+You can designate specific components for motion planning by passing in the resource name (note the use of the arm resource in the code samples below).
+The `worldState` we constructed earlier is also passed in so that the Motion Service can consider additional information when planning.
+
+The sample pose given below can be adjusted to fit your specific circumstances.
+Remember that X, Y, and Z coordinates are specified in millimeters.
+
+Again, a note:
+
+{{< alert title="Caution" color="caution" >}}
+Executing code presented after this point *will* induce motion in a connected robotic arm!
+Keep the space around the arm clear!
+{{< /alert >}}
+<br><br>
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```python {class="line-numbers linkable-line-numbers"}
+# Generate a sample "start" pose to demonstrate motion
+test_start_pose = Pose(x=510.0, y=0.0, z=526.0, o_x=0.7071, o_y=0.0, o_z=-0.7071, theta=0.0)
+test_start_pose_in_frame = PoseInFrame(reference_frame="world", pose=test_start_pose)
+
+await motion_srv.move(component_name=my_arm_resource, destination=test_start_pose_in_frame, world_state=world_state)
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+Because of ongoing experimental API changes, we must pass in a `slam` service resource name when using the Motion service to `Move`.
+Add `"go.viam.com/rdk/services/slam"` to your import list to provide access to the `slam` service.
+This tutorial will not cover any other SLAM content.
+
+```go {class="line-numbers linkable-line-numbers"}
+// Generate a sample "start" pose to demonstrate motion
+testStartPose := spatialmath.NewPose(
+  r3.Vector{X: 510.0, Y: 0.0, Z: 526.0},
+  &spatialmath.OrientationVectorDegrees{OX: 0.7071, OY: 0.0, OZ: -0.7071, Theta: 0.0},
+)
+testStartPoseInFrame := referenceframe.NewPoseInFrame(referenceframe.World, testStartPose)
+
+_, err = motionService.Move(context.Background(), myArmResource, testStartPoseInFrame, worldState, nil, slam.Named(""), nil)
+if err != nil {
+  logger.Fatal(err)
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+<!-- TODO : In the future, we should add some specific information on the importance of the frame chosen as the point of reference for PoseInFrame variables -->
+<!-- ## Managing Pose References -->
+
+## Commanding other components to move with the Motion Service
+
+This section will require you to add a new component to your robot.
+One device that is very commonly attached to the end of a robot arm is a **gripper**.
+Most robot arms pick up and manipulate objects in the world through the use of a gripper, so learning how to directly command a gripper to move is highly valuable.
+Though various Motion Service commands cause the gripper to move, ultimately the arm is doing all of the work in these situations.
+This is possible because the Motion Service considers other parts of the robot (through the [Frame System](/services/frame-system/) when calculating how to achieve the desired motion.
```

Comment:
```suggestion
This is possible because the Motion Service considers other parts of the robot (through the [Frame System](/services/frame-system/) when calculating how to achieve the desired motion).
```